web/admin/panels/Validations.js

```diff
@@ -258,25 +257,19 @@ function ValidationPanel() {
     );
   }, [entriesToValidateByWorker]);
 
-  useDeepCompareEffect(() => {
-    setUsers(adminStore.validationsFilters.users);
-  }, [adminStore.validationsFilters.users]);
-
-  const handleUserFilterChange = users => {
+  const handleUserFilterChange = newUsers => {
     const unselectedTeams = teams.map(team => ({
       ...team,
       selected: false
     }));
+    setUsers(newUsers);
+    setTeams(unselectedTeams);
     adminStore.dispatch({
       type: ADMIN_ACTIONS.updateValidationsFilters,
       payload: { teams: unselectedTeams, users: users }
```

Comment:
```suggestion
      payload: { teams: unselectedTeams, users: newUsers }
```
:red_circle: je pense que certains des bugs que j'ai decrit viennent de la